.github/workflows/linter.yml

```diff
@@ -18,9 +18,9 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Lint Code Base
-        uses: github/super-linter@v3
+      - name: Lint protobuf
+        uses: robshakir/protolint-action@v0.3.1
```

Comment:
why not use something not you for hosting?

Reply:
We can create an `openconfig` repo for this, but I'm going to submit a PR to the original owner of this action so that I can remove the fork -- it just looks like it hasn't kept pace with the releases of protolint.

Reply:
sgtm

Reply:
PR submitted upstream and now back to using the upstream action.